### PR TITLE
migrate inja to conan v2

### PR DESCRIPTION
**Pull request by AndreyMlashkin**
This conan v2 migration PR was generated by automatoc script version 10.
It uses simple find and replace technics. Feel free to extend it.

### CODE REVIEW

**Review by jgsogo — APPROVED** [review]

**Comment by prince-chrismc** [comment]
Fixing the imports is a great first step but I would not exactly call this v2 migration new generators + test_package is really the bench mark... also the min conan version needs to be bumped to ~1.45 when the tools.files was added

File: recipes/inja/all/conanfile.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import os
-from conans import ConanFile, tools
+from conan import ConanFile, tools
```
Comment (SpaceIm) [inline recipes/inja/all/conanfile.py:2]: No, it's not recommended to import tools directly

File: recipes/inja/all/test_package/conanfile.py [hunk 2]
```diff
@@ -11,6 +12,6 @@ def build(self):
         cmake.build()
 
     def test(self):
-        if not tools.cross_building(self.settings):
+        if not tools.build.cross_building(self, self.settings):
```
Comment (SpaceIm) [inline recipes/inja/all/test_package/conanfile.py:15]: wrong signature

File: recipes/inja/all/conanfile.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
 
 required_conan_version = ">=1.28.0"
```
Comment (SpaceIm) [inline recipes/inja/all/conanfile.py:4]: wrong required_conan_version

**Comment by SpaceIm** [comment]
Please read my review. Anyway I've submitted https://github.com/conan-io/conan-center-index/pull/12811. I don't trust this automated script, not reliable enough for the moment.

**Review by prince-chrismc — CHANGES_REQUESTED** [review]
Same as https://github.com/conan-io/conan-center-index/pull/12753#pullrequestreview-1093226611 🙏  which is what space wrote above

**Comment by uilianries** [comment]
Closing as superseded by #12811

**Pull request closed without merging**